packages/types/src/extension-types.ts

```diff
@@ -78,6 +78,7 @@ export enum ID {
   PAYMENT_NETWORK_ETH_FEE_PROXY_CONTRACT = 'pn-eth-fee-proxy-contract',
   PAYMENT_NETWORK_ETH_INPUT_DATA = 'pn-eth-input-data',
   PAYMENT_NETWORK_NATIVE_TOKEN = 'pn-native-token',
+  PAYMENT_NETWORK_ANY_TO_NATIVE_TOKEN = 'pn-any-to-native-token',
```

Comment:
So this one will replace `pn-eth-input-data`?

Reply:
@kevindavee the `pn-any-to-eth-proxy` is specific to ethereum (or evm-chain) while the `any-to-native` one can easily be updated to adapt to the native currency of any chain.

In the future it might be a good idea to make the `any-to-eth-proxy` an extension of the `any-to-native` for more consistency.